guides/source/developers/extensions/writing-extensions.html.md

```diff
@@ -0,0 +1,67 @@
+# Writing a New Solidus Extension
+
+Sometimes building a site powered by Solidus will require customization.
+Solidus extensions are Rails engines that allow you to extend or change the
+functionality of the Solidus platform.
```

Comment:
We should mention the Rails engine guides. As those are mandatory for extension developers.